### PR TITLE
Ability to create a space

### LINKED ISSUE #6: Ruimte aanmaken
Als een beheerder wil ik een ruimte kunnen toevoegen aan het systeem, zodat deze beschikbaar kan worden gemaakt voor reserveringen.

### Omschrijving

Binnen het KA-Huis zijn er ruimtes die door vrijwilligers gereserveerd kunnen worden. Deze ruimtes moeten binnen systeem gespecificeerd worden.

### Acceptatiecriteria

1. Er moet een naam opgegeven worden.
2. Er kan een optionele omschrijving gegeven worden.
3. De mogelijkheid om de ruimte te reserveren moet aan- en uitgezet kunnen worden.
4. Enkel gebruikers met rol beheerder mogen nieuwe ruimtes aanmaken.

### Techical tasks

- [ ] Create Inertia page view based on (https://github.com/KA-Huis/space-management-frontends/issues/2)
- [x] Create controller in admin namespace
- [x] Define routes
- [x] Validate form request
- [x] Write feature tests
- [x] Write unit tests

### Testcases

- [x] Test the create space form is shown
- [x] Test that a new space can be created

### PR DESCRIPTION
**Pull request by cyrildewit**
See #6

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,46 @@
+APP_NAME="KA-Huis"
```
Comment (cyrildewit) [inline .env]: @cyrildewit  you shouldn't commit .env file. This is due to security reasons a bad thing to do.

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -12,30 +12,28 @@ jobs:
       matrix:
```
Comment (cyrildewit) [inline .github/workflows/run-tests.yml:15]: This is currently faililng. See the build.

**Comment by cyrildewit** [comment]
I added a feature test for the `SpaceControler`.

**Comment by cyrildewit** [comment]
I finally managed to fix the broken test build. Oof.

**Comment by cyrildewit** [comment]
The dashboard page was missing a feature test, so I added one.

**Review by MiKessels — APPROVED** [review]
LGTM

Not familiar with PHP code, maybe something @joeyvonck can approve?

**Pull request merged**